--- paco-1.10.2/doc/Makefile.in	2005-11-06 20:26:22.000000000 +0100
+++ paco-1.10.2/doc/Makefile.in	2006-02-06 17:08:11.000000000 +0100
@@ -510,12 +510,7 @@
 
 
 install-data-local: pacorc
-	test -d $(sysconfdir) || $(mkinstalldirs) $(sysconfdir)
-	if test -f $(sysconfdir)/pacorc; then \
-	  $(INSTALL_DATA) pacorc $(sysconfdir)/pacorc.new; \
-	else \
-	  $(INSTALL_DATA) pacorc $(sysconfdir)/pacorc; \
-	fi
+	  $(INSTALL_DATA) pacorc $(DESTDIR)$(sysconfdir)/pacorc;
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
